Nodesets/Opc.Ua.WoT.NodeIds.csv

```diff
@@ -0,0 +1,2 @@
+AssetManagement,5001,Object
```

Comment:
This is an intermediate file from model compiler and can be removed.

Reply:
Will remove.